src/html/js/app.js

```diff
@@ -33,6 +33,53 @@ function getProfileFromSession(callback) {
   Common.getProfile(Common.getCellUrl(), callback);
 };
 
+function selectAttributes() {
+    $.ajax({
+        type: "GET",
+        url: Common.getBoxUrl() + "Engine/data?filename=interests.json",
+        headers: {
+            'Authorization': 'Bearer ' + Common.getToken(),
+            'Accept':'application/json'
+        }
+    }).done(function(data){
+        if (!_.isEmpty(data.keywords)) {
+            $('#not-change-bg-check-list li.pn-check-list').each(function(){
+                if (_.contains(data.keywords, $(this).data('keyword'))) {
+                    $(this).click();
+                }
+            });
+        }
+    });
+}
+
+function setAttributes() {
+    let currentInterests = _.map(
```

Comment:
It is better to use const instead of let. But it is not required.

Reply:
I changed to const according to your comment. Please verify.